From 153dba8911230c25b82dd216afc03408e294af26 Mon Sep 17 00:00:00 2001
From: Chris Bieneman <beanz@apple.com>
Date: Thu, 28 Apr 2016 18:22:01 +0000
Subject: [PATCH 2009/2012] [CMake] Adding some missing CMake includes. NFC.

This happens to be working now because the includes exist in another CMake file that is included before this one. That will change with upcoming refactoring.

git-svn-id: https://llvm.org/svn/llvm-project/compiler-rt/trunk@267912 91177308-0d34-0410-b5e6-96231b3b80d8
(cherry picked from commit ee9e22a7cde70075c7643f6cf854d26cfbe08a05)
---
 cmake/Modules/CompilerRTUtils.cmake | 3 +++
 1 file changed, 3 insertions(+)

diff --git llvm_release_38/projects/compiler-rt/cmake/Modules/CompilerRTUtils.cmake macports_release_38/projects/compiler-rt/cmake/Modules/CompilerRTUtils.cmake
index a8fd89a..c61c874 100644
--- llvm_release_38/projects/compiler-rt/cmake/Modules/CompilerRTUtils.cmake
+++ macports_release_38/projects/compiler-rt/cmake/Modules/CompilerRTUtils.cmake
@@ -1,3 +1,6 @@
+include(CMakePushCheckState)
+include(CheckSymbolExists)
+
 # Because compiler-rt spends a lot of time setting up custom compile flags,
 # define a handy helper function for it. The compile flags setting in CMake
 # has serious issues that make its syntax challenging at best.
-- 
2.10.1

